src/Ionide.Analyzers/Suggestion/CopyAndUpdateRecordChangesAllFieldsAnalyzer.fs

```diff
@@ -17,7 +17,9 @@ let analyze parseTree (typedTree: FSharpImplementationFileContents option) =
             { new SyntaxCollectorBase() with
                 override x.WalkExpr(_, e: SynExpr) =
                     match e with
-                    | SynExpr.Record(copyInfo = Some _; recordFields = fields) -> xs.Add(fields, e.Range)
+                    | SynExpr.Record(copyInfo = Some(synExpr, (withRange, _)); recordFields = fields) ->
+                        let fixRange = Range.unionRanges synExpr.Range (Range.shiftEnd 0 1 withRange)
```

Comment:
This `shiftEnd` is interesting, this is to get rid of the space after the `with` keyword?

Reply:
Yes, exactly. So far, it also seems to work with multiline constructs.